webhook_xblock/static/js/src/webhook_xblock.js

```diff
@@ -1,28 +1,45 @@
 /* Javascript for WebhookXblock. */
 function WebhookXblock(runtime, element) {
+    var frequency = document.getElementById('frequency').value;
+    var button_text = document.getElementById('button_text').value;
+    var handlerUrl = runtime.handlerUrl(element, 'send_payload');
 
-    function updateCount(result) {
-        $('.count', element).text(result.count);
+    function notifyStudent(result) {
+        alert('The information has been sent');
     }
 
-    var handlerUrl = runtime.handlerUrl(element, 'increment_count');
+    function checkFrequency() {
+        if (frequency != 'sent-by-student'){
+            var div = document.getElementById('center-button');
+            var button = document.createElement('button');
+            button.type = 'button';
+            button.innerHTML = button_text;
+            button.className = 'btn-styled';
+            
+            button.onclick = function() {
+                $.ajax({
+                    type: "POST",
+                    url: handlerUrl,
+                    data: JSON.stringify({'send_payload': 'True'}),
+                    success: notifyStudent
```

Comment:
even when it's not sent by the student?

Reply:
@mariajgrimaldi I think I'm going to disable this button when running from studio mode and show the user (staff) a proper message

Reply:
@mariajgrimaldi this was fixed